include/zephyr/drivers/lora.h

```diff
@@ -15,7 +15,7 @@
  * @{
  */
 
-#include <zephyr/types.h>
```

Comment:
please include `<stdint.h>`

Reply:
Is this really required when it comes in with kernel.h?

Reply:
We're including `kernel.h` here for `k_poll_signal`/`k_timeout_t`. The fact that `kernel.h` includes `stdint.h` is an implementation detail. In fact, it doesn't include it directly but via `kernel_includes.h` -> `types.h`. Unfortunately, we expose many internals because we need static allocation. If Kernel devices one day to drop usage of `stdint.h`, `lora.h` users be in trouble.

Reply:
https://google.github.io/styleguide/cppguide.html#Include_What_You_Use
https://google.github.io/styleguide/cppguide.html#Self_contained_Headers

Reply:
with `#include <stdint.h>` in the /include/zephyr/drivers/lora.h